### DISCUSSION TITLE
onDrop rejectFile callback incorrect name

### DISCUSSION THREAD

**Discussion by SuperStasSvitaminomC** [react-dropzone/react-dropzone#1365] [General]
hello, onDrop function sometimes returns an incorrect value if the file is included in the rejectFile callback 

file drop name: archive.tar.lzma

onDrop rejectFile:
{
   "path": "archive.tar.lzma0:-9:1120:44"
   "name": "archive.tar.lzma0:-9:1120:44"
}

os: ubuntu 22
brows: chrome

**Comment by rolandjitsu** [comment]
@SuperStasSvitaminomC could please try to reproduce the issue using a codesandbox (you can fork https://codesandbox.io/p/sandbox/react-dropzone-example-jq6d5 to do so)?
